### PR TITLE
Bump Catalyst dev version

### PR DESCRIPTION
**Pull request by rauletorresc**
**Context:** `main` branch needs to bump to the new dev version.

### CODE REVIEW

File: frontend/catalyst/_version.py [hunk 1]
```diff
@@ -16,4 +16,4 @@
    Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.8.0-dev18"
+__version__ = "0.9.0-dev"
```
Comment (rmoyard) [inline frontend/catalyst/_version.py:19]: No sure if it is dev or dev0?

Comment (rauletorresc) [reply frontend/catalyst/_version.py:19]: @dime10 what do you think?

Comment (dime10) [reply frontend/catalyst/_version.py:19]:
```suggestion
__version__ = "0.9.0-dev0"
```

**Review by dime10 — APPROVED** [review]
Thanks!

File: frontend/catalyst/_version.py (hunk 1 above)
Comment (rauletorresc) [reply frontend/catalyst/_version.py]: Done

**Pull request merged**